### ISSUE TITLE
shell exitcode should return 1 if vladiate fails (instead of 0)

### ISSUE THREAD

**Issue by zoobab** [di/vladiate#54]
I am calling vladiate to validate a valid data.csv file, the exitcode returned is 0 for success:

```
$ vladiate
Validating YourFirstValidator(source=LocalFile('data.csv'))
Passed! :)
$ echo $?
0
````
When I mod data.csv to make vladiate fail, the exit code is still 0 (success):

````
$ vladiate
Validating YourFirstValidator(source=LocalFile('data.csv'))
Failed :(
  SetValidator failed 1 time(s) (1.5%) on field: 'provisionstate'
    Invalid fields: ['notprovisioned2']
$ echo $?
0
````

Would it be possible to modify vladiate to return an exitcode of failure (1)?

best,

**Comment by di** [comment]
Hi @zoobab, thanks for the report! I've fixed this and released it in version [0.0.20](https://pypi.org/project/vladiate/0.0.20/). This will actually return [EX_DATAERR](https://docs.python.org/2/library/os.html#os.EX_DATAERR) if the validation fails, which is commonly exit code 65 on unix systems, but it should suffice for your purpose since it will be non-zero.

**Comment by zoobab** [comment]
Thanks again for the quick fix, I just tested it now, and it works like a charm!

Also, I will contribute some more examples, as I found the current examples not straight forward for a newcomer.
